client/jira.go

```diff
@@ -226,6 +235,21 @@ func AssignIssue(issueId, username string) error {
 	return err
 }
 
+func GetTransitions(issueId string) (TransitionsResult, error) {
+	var results TransitionsResult
```

Comment:
can we name this result 
```go
var result TransitionsResult
```

think it makes more sense since the type is a Result for Transitions